apps/api/v1/pages/api/event-types/_post.ts

```diff
@@ -311,18 +311,18 @@ async function postHandler(req: NextApiRequest) {
 }
 
 async function checkPermissions(req: NextApiRequest) {
-  const { isAdmin } = req;
+  const { isSystemWideAdmin } = req;
   const body = schemaEventTypeCreateBodyParams.parse(req.body);
   /* Non-admin users can only create event types for themselves */
-  if (!isAdmin && body.userId)
+  if (!isSystemWideAdmin && body.userId)
```

Comment:
I tried creating an event type for org user and it is not showing up in /event-types

![Screenshot 2024-05-10 at 2 28 36 PM](https://github.com/calcom/cal.com/assets/53316345/a9ae1c27-ae8d-4041-b94d-7770db098a5d)
![Screenshot 2024-05-10 at 2 28 52 PM](https://github.com/calcom/cal.com/assets/53316345/662ecb22-14ed-40b8-834a-fc120800e9d0)

Reply:
The newly created event Type does not contain `Profile` column in event type

Reply:
Org Admin is not yet implemented for event-types. It is only implemented for /bookings endpoints as a starting point